.github/workflows/release_tests.yml

```
@@ -0,0 +1,89 @@
+
+      - name: Install System Dependencies
+        run: |
+          sudo apt-get update
+          sudo apt install python3-dev swig libssl-dev portaudio19-dev libpulse-dev libfann-dev
+
+      - name: Install test dependencies
+        run: |
+          pip install -r requirements/tests.txt
+
+      - name: Install core repo with constraints
+        run: |
```

actionlint

shellcheck reported issue in this script: SC2102:info:1:17: Ranges can only match single chars (mentioned due to duplicates)

(shellcheck)